webui/src/App.js

```diff
@@ -18,8 +22,11 @@ function App() {
 
   return initialized ? (
     <BrowserRouter>
+    <NavBar />
       <Routes>
         <Route path="/" element={<Home />} />
+        <Route path="/Pools" element={<PoolCreator />} />
+        <Route path="/Swap" element={<Swap />} />
```

Comment:
I think `path=/pools` and `path=/swap` will be better.

Reply:
Ok, I will correct that. Thanks for the review.